blueocean-personalization/src/test/js/redux/FavoritesStore-spec.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Created by cmeyers on 7/11/16.
+ */
+import { assert } from 'chai';
+
+import Store from '../../../main/js/redux/FavoritesStore';
+import { ACTION_TYPES, FavoritesState } from '../../../main/js/redux/FavoritesStore';
+
+const getDefaultFavorites = () =>
+    JSON.parse(`
+        [
+            {
+                "_class": "io.jenkins.blueocean.service.embedded.rest.FavoriteImpl",
+                "_links": {
+                    "self": {
+                        "_class": "io.jenkins.blueocean.rest.hal.Link",
+                        "href": "/blue/rest/users/cmeyers/favorites/blueocean%2FUX-301/"
+                    }
+                },
+                "item": {
+                    "_class": "io.jenkins.blueocean.service.embedded.rest.BranchImpl",
+                    "_links": {
+                        "self": {
+                            "_class": "io.jenkins.blueocean.rest.hal.Link",
+                            "href": "/blue/rest/organizations/jenkins/pipelines/blueocean/branches/UX-301/"
+                        }
+                    }
+                }
+            },
+            {
+                "_class": "io.jenkins.blueocean.service.embedded.rest.FavoriteImpl",
+                "_links": {
+                    "self": {
+                        "_class": "io.jenkins.blueocean.rest.hal.Link",
+                        "href": "/blue/rest/users/cmeyers/favorites/jenkinsfile-experiments%2Fmaster/"
+                    }
+                },
+                "item": {
+                    "_class": "io.jenkins.blueocean.service.embedded.rest.BranchImpl",
+                    "_links": {
+                        "self": {
+                            "_class": "io.jenkins.blueocean.rest.hal.Link",
+                            "href": "/blue/rest/organizations/jenkins/pipelines/jenkinsfile-experiments/branches/master/"
+                        }
+                    }
+                }
+            }
+        ]
+    `);
```

Comment:
I was trying to avoid some tedious formatting to make it lint-compliant. I am not sure if I could have "ignored next line" in lint rules and if it would ignore the entire expression, or just the next time :P At any rate I cleaned this up in 9beb47a of my next branch.